scripts/vercel-ignore-build-step.sh

```diff
@@ -1,13 +1,3 @@
 #!/bin/bash
 
-echo "VERCEL_GIT_COMMIT_AUTHOR_LOGIN: $VERCEL_GIT_COMMIT_AUTHOR_LOGIN"
-
-if [ "$VERCEL_GIT_COMMIT_AUTHOR_LOGIN" == "renovate-bot" ]; then
-  # Don't build
-  echo "🛑 - Renovate Build cancelled"
-  exit 0;
-else
-  # Proceed with the build
-    echo "✅ - Build can proceed"
-  exit 1;
-fi
+exit 1;
```

Comment:
@n1ru4l maybe we can just remove this script entirely ? 🧐

Reply:
to be honest I don't know where it is referenced from 😅

Reply:
Seems like it is related to configuration on the vercel app: https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel

Reply:
maybe @charlypoly knows

Reply:
I found it and removed the script reference in here;
![image](https://user-images.githubusercontent.com/20847995/172151978-13524c55-223c-4400-9a4b-12b873bc9044.png)